api/Console.json

```diff
@@ -289,6 +289,104 @@
           }
         }
       },
+      "debug": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/debug",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "4"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "substitution_strings": {
+          "__compat": {
+            "description": "Substitution strings",
+            "support": {
+              "chrome": {
+                "version_added": true,
+                "notes": "In version 28, if a negative value is passed to %d, it will be rounded down to the closest negative integer, so -0.1 becomes -1."
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "9"
+              },
+              "firefox_android": {
+                "version_added": "9"
+              },
+              "ie": {
+                "version_added": "10",
+                "notes": "%c is not supported, %d will render as 0 when it is not a number"
```

Comment:
I think this ought to be an array (and the notes themselves formatted a little bit). Like this, maybe:

```
"notes": [
  "<code>%c</code> is not supported.",
  "<code>%d</code> renders as <code>0</code> when it is not a number."
]
```